fs-storage/src/base_storage.rs

```diff
@@ -0,0 +1,27 @@
+use data_error::Result;
+use std::collections::BTreeMap;
+
+pub trait BaseStorage<K, V>: AsRef<BTreeMap<K, V>> {
+    fn set(&mut self, id: K, value: V);
```

Comment:
I think we should have `get` too here
See https://github.com/ARK-Builders/ark-rust/pull/23#discussion_r1570703803

Reply:
https://github.com/ARK-Builders/ark-rust/pull/23#discussion_r1573135363